qgis4/ansible/README.md

```diff
@@ -0,0 +1,9 @@
+# Testing with vagrant
```

Comment:
Can you add this in `qgis4/README.md` within the deployment part instead please?